dali/test/python/test_operator_decoders_image.py

```diff
@@ -47,19 +48,21 @@ def get_img_files(data_path, subdir='*', ext=None):
         txt_files = glob.glob(data_path  + f"/{subdir}/*.txt")
         return list(set(files) - set(txt_files))
 
-class DecoderPipeline(Pipeline):
-    def __init__(self, data_path, batch_size, num_threads, device_id, device, use_fast_idct=False, memory_stats=False):
-        super(DecoderPipeline, self).__init__(batch_size, num_threads, device_id, prefetch_queue_depth=1)
-        self.input = ops.readers.File(file_root = data_path,
-                                      shard_id = 0,
-                                      num_shards = 1)
-        self.decode = ops.decoders.Image(device = device, output_type = types.RGB, use_fast_idct=use_fast_idct,
-                                         memory_stats=memory_stats)
+@pipeline_def
+def create_decoder(data_path, device, use_fast_idct=False, memory_stats=False):
```

Comment:
```suggestion
def decoder_pipe(data_path, device, use_fast_idct=False, memory_stats=False):
```
would be a more descriptive name IMO

Reply:
Done